app/services/usage-statistics.ts

```diff
@@ -142,14 +155,16 @@ export class UsageStatisticsService extends Service {
 
     this.anaiticsEvents.length = 0;
 
-    const request = new Request(
-      `https://${this.hostsService.streamlabs}/api/v5/analytics/slobs/ping`,
-      {
-        method: 'post',
-        headers: authorizedHeaders(this.userService.apiToken),
-        body: JSON.stringify(data || {}),
-      },
-    );
+    const request = new Request(`https://${this.hostsService.analitycs}/slobs/data/ping`, {
+      headers,
+      method: 'post',
+      body: JSON.stringify(data || {}),
+    });
     fetch(request).then(handleResponse);
   }
+
+  @mutation()
+  private SET_UUID() {
+    this.state.uuid = uuid();
```

Comment:
I think we should back this out per above comment, but I just wanted to let you know this is unsafe.  Mutations need to deterministic, so running them in multiple windows yields the same result.  This will result in child windows having a different UUID than the main window.

Reply:
Good catch